src/hooks/usePassword.ts

```diff
@@ -0,0 +1,20 @@
+import { useState, useMemo } from 'react'
+
+export function usePassword() {
+  const [value, setValue] = useState('')
+  const [touched, setTouched] = useState(false)
+
+  const error = useMemo(() => {
+    if (!value) return 'PW를 입력해주세요.'
+    if (value.length < 8) return 'PW는 최소 8글자 이상이어야 합니다.'
+    return ''
+  }, [value])
+
+  return {
+    value,
+    change: setValue,
+    touched,
+    error,
+    onBlur: () => setTouched(true),
+  }
+}
```

Comment:
잘 보면 형식이 useEmail과 굉장히 유사한걸 알 수 있어요 ㅎㅎ
이렇게 표현해줄 수 있지 않을까요~?

```suggestion
export function usePassword() {
  return useInput(validatePassword)
}
```

큰 틀은 useInput으로 만들어주고, 내부의 오류메세지를 만드는 로직만 validatePassword로 지정해주는거죠.